Gemfile

```diff
@@ -12,6 +12,7 @@ gem 'foundation-icons-sass-rails'
 gem 'foundation-rails'
 gem 'jquery-rails'
 gem 'sass-rails', '~> 4.0.0'
+gem 'scss-lint'
```

Comment:
@rustemginiyatullin looks like `scss-lint` should be used only for developent/test envs. Please move in to the appropriate section in the `Gemfile`.